phpBB/adm/style/acp_profile.html

```diff
@@ -17,7 +17,7 @@
 		</div>
 	<!-- ENDIF -->
 
-	<form id="add_profile_field" method="post" action="{U_ACTION}">
+	<form id="add_profile_field" method="post" action="{U_ACTION}"{S_FORM_ENCTYPE}>
```

Comment:
By the way, if you want to continue discussion, please, let's use a more appropriate way:
https://area51.phpbb.com/phpBB/viewtopic.php?f=126&t=48001